db/migrate/20171230160309_rename_recreate_publishers.rb

```diff
@@ -33,5 +33,6 @@ def change
       t.index :pending_email
       t.index :created_at
     end
+    change_column_default :legacy_publishers, :verified, false
```

Comment:
Don't include this migration.

Reply:
Ok I created a new migration for it, we need to ensure this null property is enforced.

Reply:
The changes to that column isn't relevant because it'll be changing the `legacy_publishers` table which we don't use.